File: Program/php/Kendaraan.php
<?php
// Base Class
class Kendaraan {
    protected $id;
    protected $merk;
    protected $tahun;
    protected $foto;

    public function __construct($id = 0, $merk = "", $tahun = 0, $foto = "") {
        $this->id = $id;
        $this->merk = $merk;
        $this->tahun = $tahun;
        $this->foto = $foto;
    }

    public function getId() { return $this->id; }
    public function setId($id) { $this->id = $id; }

    public function getMerk() { return $this->merk; }
    public function setMerk($merk) { $this->merk = $merk; }

    public function getTahun() { return $this->tahun; }
    public function setTahun($tahun) { $this->tahun = $tahun; }

    public function getFoto() { return $this->foto; }
    public function setFoto($foto) { $this->foto = $foto; }
}
?>

File: Program/python/Kendaraan.py
# Base class
class Kendaraan:
    def __init__(self, id_kendaraan=0, merk="", tahun=0):
        # Atribut private / encapsulated menggunakan prefix underscore
        self._id = id_kendaraan
        self._merk = merk
        self._tahun = tahun

    # Getter dan Setter untuk ID
    def get_id(self):
        return self._id

    def set_id(self, id_kendaraan):
        self._id = id_kendaraan

    # Getter dan Setter untuk Merk
    def get_merk(self):
        return self._merk

    def set_merk(self, merk):
        self._merk = merk

    # Getter dan Setter untuk Tahun
    def get_tahun(self):
        return self._tahun

    def set_tahun(self, tahun):
        self._tahun = tahun